docs/processes/qualityassurance/git-gontribution-guidelines.md

```diff
@@ -0,0 +1,224 @@
+  │
+  └─⫸ Commit Type: build|ci|docs|feat|fix|perf|refactor|test
+```
+
+The \<type\> and \<summary\> fields are mandatory, the (\<scope\>) field is optional.
+
+The \<type\> must be one of the following:
+
+- **build** : Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- **ci** : Changes to our CI configuration files and scripts (example scopes: Circle, BrowserStack, SauceLabs)
+- **docs** : Documentation only changes
+- **feat** : A new feature
+- **fix** : A bug fix
+- **perf** : A code change that improves performance
+- **refactor** : A code change that neither fixes a bug nor adds a feature
+- **test** : Adding missing tests or correcting existing tests
+
+Doubtfire-LMS recommends Chris Beam's post on [How to Write Good Commit Messages](http://chris.beams.io/posts/git-commit/) to improve your commit message writing.
+
+**Use the imperative mood in your commit subject line**
+
+Write your commits in the imperative mood and not the indicative mood
+
+- "Fix a bug" and **not** "Fix*ed* a bug"
+- "Change the behaviour of Y" and **not** "_Changed_ the behaviour of Y"
+- "Add new API methods" and **not** "Sweet new API methods"
+
+Keep the subject line (top line) concise; keep it **within 50 characters**.
+
+Use the body (lines after the top line) to explain why and what and _not_ how; keep it **within 72 characters**.
+
+### But how can I write new lines if I'm using `git commit -m "Message"`?
+
+Don't use the `-m` switch. Use a text editor to write your commit message instead.
+
+If you are using the command line to write your commits, it is useful to set your git editor to make writing a commit body easier. You can use the following command to set your editor to Visual Studio Code, `nano`, `emacs`, `vim`, `atom`.
+
+```bash
+git config --global core.editor "code --wait"
+git config --global core.editor nano
+git config --global core.editor emacs
+git config --global core.editor vim
+git config --global core.editor "atom --wait"
+```
+
+# Git Workflow Summary
+
+## Start a new piece of work
+
+1. Synch repo and set up for new feature branch (remember to use Thoth Tech [branching guidelines](#__Branching_Guide) for naming the new branch):
+
+```bash
+git checkout main
+git pull main
+git checkout -b feature/\<project\_subtask\_description\>
+```
+
+2. Make changes, commit (using comments format that follows [commit guidelines](#_Commit_guidelines)) and push to origin:
+
+```bash
+git add .
+git commit
+git push -u origin feature/\<project\_subtask\_description\>
+```
+
+3. Remember to submit a [draft pull request](#draft-pull-request) via GitHub to allow for code review (and mark as ready to submit when ready to merge your changes to main).
+
+## Continue a piece of work
+
+1. If the branch has already been created in the remote repository.
+
+```bash
+git checkout feature/\<project\_subtask\_description\>
```

Comment:
**suggestion:** Similar to the suggestion above ☝🏼, could we use an example branch name to avoid escaping?